cli/prompt.go

```diff
@@ -71,24 +71,21 @@ func (h *Handler) PromptAsset(label string, allowNative bool) (ids.ID, error) {
 			if allowNative && input == symbol {
 				return nil
 			}
-			_, err := ids.FromString(input)
-			return err
+			_ = codec.LIDFromString(label, input)
```

Comment:
I don't think this is right

Reply:
You should be decoding the entire input as cb58, don't include label here? https://github.com/ava-labs/avalanchego/blob/67a8b10abd9cf8644e819c0b28f7b25894cec7fc/ids/id.go#L42